person.rb

```diff
@@ -1,6 +1,6 @@
 require './corrector'
 class Person
-  attr_reader :id
+  attr_reader :id, :rental
   attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] You should have a  method for adding rentals  for your `Person` class, the person itself should be sent as a parameter to create the new rental achieving the `has-many` association`.
Your code should look something like this :point_down: 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```